pkgs/development/libraries/glog/default.nix

```diff
@@ -20,9 +20,18 @@ stdenv.mkDerivation rec {
     })
   ];
 
+  postPatch = lib.optionalString stdenv.isDarwin ''
+    # A path clash on case-insensitive file systems blocks creation of the build directory.
+    # The file in question is specific to bazel and does not influence the build result.
+    rm BUILD
+  '';
+
   nativeBuildInputs = [ cmake ];
 
-  cmakeFlags = [ "-DBUILD_SHARED_LIBS=${if static then "OFF" else "ON"}" ];
+  buildInputs = [ gflags ];
```

Comment:
Perhaps should use the `propagatedBuildInputs`:

```
builder for '/nix/store/2sh6ayyd27csmp89dmhj52inca6cnl97-ceres-solver-1.14.0.drv' failed with exit code 2; last 10 log lines:
  In file included from /private/var/folders/1f/2mj_ng1n2y3_wc59fgw3fn_r0000gn/T/nix-build-ceres-solver-1.14.0.drv-0/ceres-solver-1.14.0/internal/ceres/block_random_access_sparse_matrix.cc:31:
  In file included from /private/var/folders/1f/2mj_ng1n2y3_wc59fgw3fn_r0000gn/T/nix-build-ceres-solver-1.14.0.drv-0/ceres-solver-1.14.0/internal/ceres/block_random_access_sparse_matrix.h:46:
  In file included from /private/var/folders/1f/2mj_ng1n2y3_wc59fgw3fn_r0000gn/T/nix-build-ceres-solver-1.14.0.drv-0/ceres-solver-1.14.0/internal/ceres/small_blas.h:40:
  /nix/store/vl1ab3177z3239b9m4va31d8s8jrgqxx-glog-0.4.0/include/glog/logging.h:86:10: fatal error: 'gflags/gflags.h' file not found
  #include <gflags/gflags.h>
           ^~~~~~~~~~~~~~~~~
  1 error generated.
```

The dependency was hardcoded into the logging.h:
```
#if 1
#include <gflags/gflags.h>
#endif
```

Reply:
Thanks for noticing.